src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1661,13 +1662,17 @@ private LocalState VisitSwitchHeader(BoundSwitchStatement node)
 
         private void VisitSwitchBlock(BoundSwitchStatement node)
         {
+            var afterSwitchState = UnreachableState();
```

Comment:
That would be a bit out of place in this source file (nearly every line could have such a comment). Every statement is unreachable unless there is some way of reaching it.